templates/celery.conf.j2

```diff
@@ -0,0 +1,9 @@
+; {{ ansible_managed }}
+
+[program:celery]
+autostart=true
+autorestart=true
+command = {{ crosshair_virtualenv }}/bin/celery_start
+user = celery
+stdout_logfile = {{ crosshair_celery_log }}
+redirect_stderr = true
```

Comment:
I _think_ you still need `stderr_logfile=NONE` or it will create a stderr_logfile with nothing in it.